src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1580,6 +1580,8 @@ void uct_dc_mlx5_iface_set_ep_failed(uct_dc_mlx5_iface_t *iface,
     ucs_status_t status;
     ucs_log_level_t log_lvl;
 
+    uct_dc_mlx5_ep_pending_purge_internal(
```

Comment:
why do we need additional function? Can we just use a callback which would remove only internal reqs?